src/hw29/homework/src/main/java/org/DatabaseConnector.java

```diff
@@ -0,0 +1,32 @@
+package org;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseConnector {
+    String url = "jdbc:mysql://localhost:3306/company";
+    String username = "root";
+    String password = "htc2009noch";
```

Comment:
For local usage as this hw better to use simpla password as example otherwise github is complaining